bounce/server/api/util.py

```diff
@@ -0,0 +1,141 @@
+"""Utility functions unsed by Bounce endpoints."""
+
+import hashlib
+import re
+from datetime import datetime, timedelta
+
+import bcrypt
+from jose import jwt
+
+# Regexes for validating passwords and usernames
+# pylint: disable=anomalous-backslash-in-string
+UPPERCASE = re.compile('[A-Z]+')
+LOWERCASE = re.compile('[a-z]+')
+NUMBERS = re.compile('[0-9]+')
+PASSWORD_SYMBOLS = re.compile('[\.\-!@#$%^&*?_+ ]+')
+USERNAME_SYMBOLS = re.compile('[\.\-_]+')
```

Comment:
is there no library you can use for user/pass validation?

Reply:
There probably is but I haven't found it.